src/test/scala/com/microsoft/hyperspace/TestUtils.scala

```diff
@@ -55,18 +55,17 @@ object TestUtils {
    * @param path Path to the folder containing files.
    * @param pattern File name pattern to delete.
    * @param numFilesToDelete Number of files to delete.
-   * @return Paths to the deleted file.
+   * @return Deleted files' FileStatus.
    */
-  def deleteFiles(path: String, pattern: String, numFilesToDelete: Int): Seq[Path] = {
+  def deleteFiles(path: String, pattern: String, numFilesToDelete: Int): Seq[FileStatus] = {
     val pathToDelete = new Path(path, pattern)
-    val fileNames = pathToDelete
+    val files = pathToDelete
       .getFileSystem(new Configuration)
       .globStatus(pathToDelete)
-      .map(_.getPath)
 
-    assert(fileNames.length >= numFilesToDelete)
-    val filesToDelete = fileNames.take(numFilesToDelete)
-    filesToDelete.foreach(FileUtils.delete(_))
+    assert(files.length >= numFilesToDelete)
+    val filesToDelete = files.take(numFilesToDelete)
+    filesToDelete.foreach(f => FileUtils.delete(f.getPath))
```

Comment:
nit: _.getPath ?

Reply:
Yes, I had tried it before but it does not work in this case (fails with below error). I guess because of being an arg for FileUtils.delete(...).

`Type mismatch, expected: FileStatus => NotInferedU, actual: Unit`